device/core/test/_internal_client_test.js

```diff
@@ -282,15 +282,6 @@ var ModuleClient = require('../lib/module_client').ModuleClient;
         });
       });
 
-      it('calls the callback immediately if the client is already disconnected', function (testCallback) {
```

Comment:
>it('calls the callback immediately if the client is already disconnected', function (testCallback) { [](start = 6, length = 100)

Why was this deleted?

Reply:
I wrote an explanation before, please take a look at the comment in the first iteration.

Reply:
Done

---
In reply to: [222774298](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222774298) [](ancestors = 222774298)